schemas/2015-05-21-preview/Microsoft.DevTestLab.json

```diff
@@ -21,13 +21,13 @@
       },
       "required": [ "type", "apiVersion", "location" ]
     },
-    "environments": {
-      "description": "Required properties to create environment in an existing DevTest Lab",
+    "virtualMachines": {
```

Comment:
Service already support the virtualMachines resource type and environments has been removed. We can take breaking changes as we are still not GA. We announced that breaking change, and contacted the few customers who had not made changes.